Week2/todo_app/program.js

```diff
@@ -21,39 +22,96 @@ class TodoModel {
         });
     }
 
-    create(description, callback) {
-        // Write code and query to create a new TODO item
+    create(user_id, description, callback) {
+        const createTodoItem = `INSERT INTO todo_items(text, is_completed, user_id)
+        VALUES(${dbConnection.escape(description)}, 0, ${dbConnection.escape(user_id)})`;
```

Comment:
Good to use escaping. There's a even neater way of escaping with `?`s - you can find it under the **Escaping query values** section in `mysql.js` official documentation :)